airbyte-webapp/src/components/connectorBuilder/useManifestToBuilderForm.ts

```diff
@@ -203,18 +200,18 @@ function manifestStreamSlicerToBuilder(
   }
 
   if (manifestStreamSlicer.type === "DatetimeStreamSlicer") {
-    const datetimeStreamSlicer = manifestStreamSlicer as DatetimeStreamSlicer;
+    const datetimeStreamSlicer = manifestStreamSlicer;
```

Comment:
:information_source: I believe `pnpm` pulled now in a newer version of `orval` which generates the types slightly better, so this isn't needed anymore.